packages/fakta-medlemskap/src/v3/components/situasjon/situasjonUtils.ts

```diff
@@ -0,0 +1,71 @@
+import { MedlemskapV3, UtlandsoppholdPeriode } from '@navikt/fp-types';
+import { getKodeverknavnFn, KodeverkType, opplysningAdresseType } from '@navikt/fp-kodeverk';
+import { alleKodeverk } from '@navikt/fp-storybook-utils';
+import kodeverkTyper from '@navikt/fp-kodeverk/src/kodeverkTyper';
+import { IntlShape } from 'react-intl';
+import { TIDENES_ENDE } from '@navikt/ft-utils';
+
+export function sorterPerioder<T extends { fom: string; tom: string }>(a: T, b: T) {
```

Comment:
Kva oppnår ein eigentleg med å bruka generics her? Kun at ein slepp å skriva "periode-typen" to gonger? Syns i sofall at det er litt overkill og gjer koden vanskelegare å lesa

Reply:
den brukes til å sortere fler objekter som har fom og tom , men funker uten generic også, så jeg dropper det da